src/benchmark.cc

```diff
@@ -17,7 +17,9 @@
 #include "internal_macros.h"
 
 #ifndef BENCHMARK_OS_WINDOWS
+#ifndef __Fuchsia__
```

Comment:
can you define a BENCHMARK_OS_FUSCHIA alongside the BENCHMARK_OS_XXX macros elsewhere? that'll help be consistent and track any other per-OS changes.

Reply:
Ok, done.